packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js

```diff
@@ -1,15 +1,15 @@
 /* eslint-disable */
-import {connect as Cerebral} from 'cerebral/react';
-import {limitedList} from "./limitedList";
-import {anotherComputed} from "./anotherComputed";
+import {connect as Cerebral} from "cerebral/react"
```

Comment:
If the fixtures have to look exactly like the resulted code we should fix this by hand.
Maybe after this PR is merged.